### PR TITLE
Auxmos

### PR DESCRIPTION
**Pull request by Absolucy**
This replaces extools with auxtools, and monstermos with katmos (which acts pretty much the same as monstermos, but multithreaded, faster, and less buggy)

**_THIS NEEDS A BUNCH OF TESTING AND REVIEW!!!_**

🆑 Lucy, CoffeeKat
rscadd: ported Katmos. atmospherics should be mostly the same, but a bit less buggy and a bit faster now.
/🆑

### CODE REVIEW

**Comment by ToasterBiome** [comment]
did you fix the weird lag

**Comment by Absolucy** [comment]
idk

**Comment by Absolucy** [comment]
@ToasterBiome

https://user-images.githubusercontent.com/65794972/154178640-279347e8-ec29-4c72-a18c-a9d2507609dd.mp4

**Comment by monster860** [comment]
I'm glad to see that my shitcode is being improved.

**Comment by Moja10** [comment]
How horrifying, I like it

**Review by patpol4 — APPROVED** [review]
Maps fine

**Comment by Altoids1** [comment]
Aww shit, here we go again.

**Review by Altoids1 — COMMENTED** [review]
If this is merged, one thing I would really like to have is a Yogstation org fork of this specific katmos & auxtools so that we are not trusting this binary (with no offense to Lucy) and so I have a consistent binary target to modify as necessary for my own João PR.

File: code/modules/atmospherics/gasmixtures/gas_mixture.dm [hunk 1]
```diff
@@ -7,41 +7,112 @@ What are the archived variables for?
...
+		dummy["TOTAL MOLES"] = total_moles()
+		dummy["VOLUME"] = return_volume()
+		dummy["THERMAL ENERGY"] = thermal_energy()
+		return debug_variable("gases (READ ONLY)", dummy, 0, src)
+
 /*
-/datum/gas_mixture/Del()
-	__gasmixture_unregister()
-	. = ..()*/
+/datum/gas_mixture/vv_get_dropdown()
```
Comment (adamsong) [inline code/modules/atmospherics/gasmixtures/gas_mixture.dm:62]: Why is this commented out?

Comment (Absolucy) [reply code/modules/atmospherics/gasmixtures/gas_mixture.dm:62]: Huh, I thought we didn't have the new vv code. ~~I guess we do.~~ Nevermind, we don't have `vv_do_topic` though.

**Comment by alexkar598** [comment]
> If this is merged, one thing I would really like to have is a Yogstation org fork of this specific katmos & auxtools so that we are not trusting this binary (with no offense to Lucy) and so I have a consistent binary target to modify as necessary for my own João PR.

Already did that when the first pr was opened.
Speaking of, @Absolucy , any new changes to auxtools?

**Comment by Absolucy** [comment]
Not sure... I'll check later

**Comment by JamieD1** [comment]
Hi Lucy, this time we can try and get this PR merged, however I have a few questions/requests.

First I would like to generalise the name, im fed up of everyone adding their name to something which has been in the game for years. Perhaps Auxmos instead or im up for suggestions.

Second, will you be around to help with any testing and bug fixing, I know its not yours and you are porting it but access to the PR would be greatly appreciate to facilitate good testing.

Third, From the video it does look awfully similar to ours, but through testing we can make sure it fits our players and server before I will commit to a full merge. For example when we can have several rounds in a row either without people noticing or when it no longer becomes an issue or a thing of question or blame. We will need to test several different game modes as well to make sure that there are no niche cases to identify.

Finally, thanks for porting this and putting in the effort, hopefully this will be a good step in the future of the archaic atmos system of SS13.

**Comment by patpol4** [comment]
Maybe this might fix the icemoon atmos issues

**Comment by adamsong** [comment]
GitHub appears to have auto closed this due to a force push. I am reopening this under that assumption, feel free to close again if I am mistaken.

**Comment by Absolucy** [comment]
I accidentally did a `git reset --hard upstream/master` on this branch and force-pushed, so I had to reflog+reset then force push again. Sorry!

**Comment by JamieD1** [comment]
You forgot the linux.

**Comment by JamieD1** [comment]
Test merge in progress

**Comment by ToasterBiome** [comment]
tritium doesn't burn in auxmos, so it piles up and plasma fires only reach like 3000K

**Comment by TheRyeGuyWhoWillNowDie** [comment]
No gasses?

**Comment by KittyNoodle** [comment]
vents always emit gas during breaches, also the pirate ship is fucked

**Comment by JamieD1** [comment]
Hows this going, we stuck?

**Comment by Absolucy** [comment]
> Hows this going, we stuck?

@JamieD1 should be fine now

**Comment by JamieD1** [comment]
Test-Merged

**Comment by Absolucy** [comment]
I'll probably never get this to work, anyways.

**Comment by Absolucy** [comment]
If I can figure out why there were problems on the server, I'll reopen this.

**Comment by ToasterBiome** [comment]
> If I can figure out why there were problems on the server, I'll reopen this.

anything I can do to help diagnose the problem?

**Comment by Absolucy** [comment]
Not really, sorry. This is mostly @jupyterkat's thing - I've just been trying to follow their instructions and telling them when shit isn't working.

**Pull request closed without merging**